lib/cn/kvset_split.c

```diff
@@ -3,6 +3,8 @@
  * Copyright (C) 2022 Micron Technology, Inc.  All rights reserved.
  */
 
+#include <sys/mman.h>
```

Comment:
`sys/mman` here and `hse_util/mman` everywhere else?

Reply:
I don't understand the point of hse_util/mman.h. The one definition is only used one time.

Reply:
Ok, I get it. `MADV_FREE` was added in Linux 4.5. Luckily, no system we support uses Linux 4.5 anymore. Probably an artifact of when you guys used to test on RHEL 6 or 7 if I had to guess.

Reply:
Removed in latest push.